client-src/index.js

```diff
@@ -66,7 +66,7 @@ self.addEventListener('beforeunload', () => {
 if (typeof window !== 'undefined') {
   const qs = window.location.search.toLowerCase();
 
-  options.hotReload = qs.indexOf('hotreload=false') === -1;
+  options.hotReload = !qs.includes('hotreload=false');
```

Comment:
Don't no use modern function in client due some developers still need IE 11 support and other old browsers https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
but we transpile them using babel right?

Reply:
Yes, but it is increase our bundle, ideally we should keep this small as possible

Reply:
gotcha lets change this then

Reply:
startsWith will also come after transpile, remove it too? I think it won't affect the bundle that much as polyfill will be really small..

Reply:
Yes, they are small, but why do this if we can use alternatives and not increase the size at all

Reply:
makes sense, should I leave change in `Server.js` or close?